src/main/java/org/fcrepo/importexport/common/Config.java

```diff
@@ -386,28 +405,29 @@ public static Config fromFile(final Map<String, String> configVars) throws Parse
             } else if (entry.getKey().equalsIgnoreCase("rdfLang")) {
                 c.setRdfLanguage(entry.getValue());
             } else if (entry.getKey().trim().equalsIgnoreCase("binaries")) {
-                if (entry.getValue().equalsIgnoreCase("true") || entry.getValue().equalsIgnoreCase("false")) {
-                    c.setIncludeBinaries(Boolean.parseBoolean(entry.getValue()));
-                } else {
-                    throw new ParseException(String.format(
-                        "binaries configuration parameter only accepts \"true\" or \"false\", \"{}\" received",
-                        entry.getValue()), lineNumber);
-                }
+                c.setIncludeBinaries(parseBoolean(entry, "binaries", lineNumber));
+            } else if (entry.getKey().trim().equalsIgnoreCase("overwriteTombstones")) {
+                c.setIncludeBinaries(parseBoolean(entry, "overwriteTombstones", lineNumber));
             } else if (entry.getKey().trim().equalsIgnoreCase("external")) {
-                if (entry.getValue().equalsIgnoreCase("true") || entry.getValue().equalsIgnoreCase("false")) {
-                    c.setRetrieveExternal(Boolean.parseBoolean(entry.getValue()));
-                } else {
-                    throw new ParseException(String.format(
-                        "external configuration parameter only accepts \"true\" or \"false\", \"{}\" received",
-                        entry.getValue()), lineNumber);
-                }
+                c.setIncludeBinaries(parseBoolean(entry, "external", lineNumber));
             } else {
                 throw new ParseException(String.format("Unknown configuration key: {}", entry.getKey()), lineNumber);
             }
         }
         return c;
     }
 
+    private static boolean parseBoolean(final Map.Entry<String, String> entry, final String key,
```

Comment:
Minor suggestion, the redundant `entry.getValue()` calls in this method could be eliminated by passing in the entry value into this method.